app/services/katello/pulp3/repository.rb

```diff
@@ -65,7 +65,7 @@ def list(args)
 
       def delete(href = repository_reference.repository_href)
```

Comment:
Looks like this still isn't being called from the orchestration::repository::Delete  action (via a distribution::Delete action)

Reply:
Unless I missed it, in which case ignore :)

Reply:
No, I did leave it out. I recall mentioning the delete distributions by path method, but apparently I didn't follow through. :(